### PR TITLE
Docstrings for plotting.py

### PR DESCRIPTION
**Pull request by snilsn**
This PR contains revised docstrings for the plotting.py functions I've gotten to work so far.

Unfortunately, this time there are more functions which are not covered yet. To keep track of that I use this [table](https://docs.google.com/spreadsheets/d/1dD7TFlRgcMKWeh0NkHuj24O6guvbEdTrm0x2eCOMtnM/edit#gid=1309608279)

I created a [notebook](https://github.com/snilsn/tobac_functions/blob/master/plotting_functions.ipynb) of my experiments with the functions that are marked as 'broken?'. I suspect some of them to have bugs, maybe resulting from changes in other parts of tobac. Others I perhaps use in the wrong way. None of these I saw in use in any of the examples.

This is probably worth a separate issue, like suggested in #146. We should discuss whether they should still be included in this PR.

* [x] Have you followed our guidelines in CONTRIBUTING.md? 
* [x] Have you self-reviewed your code and corrected any misspellings? 
* [x] Have you written documentation that is easy to understand?
* [x] Have you written descriptive commit messages? 
* [x] Have you formatted your code using black? 
* [x] Have all tests passed in your local clone? 
* [x] Have you kept your pull request small and limited so that it is easy to review? 
* [x] Have the newest changes from this branch been merged? 

Not needed:

* [ ] Have you added NumPy docstrings for newly added functions?
* [ ] If you have introduced a new functionality, have you added adequate unit tests?
* [ ] If you have introduced a new functionality, have you added an example notebook?

### CODE REVIEW

**Comment by freemansw1** [comment]
Thanks Nils! I've requested @JuliaKukulies and myself for review. I'll try to get this reviewed before the developers' meeting. Agreed that we should discuss the plotting.py functions; I had no idea that this many of them were broken. I'm happy for you to open the issue, or I can do that!

**Review by JuliaKukulies — CHANGES_REQUESTED** [review]
Thank you @snilsn for these comprehensive documentation improvements! This was for sure not an easy task, and based on your notebooks it seems like the plotting functions need some general revisions, which probably will require some update of these docstrings at a later point. I would, however, be happy to get these changes in for v1.4 as these are a nice complement to your PR #138. Again, just a few comments and suggestions, mainly typos and some clarifications

File: tobac/plotting.py
Comment (snilsn) [reply tobac/plotting.py:3]: Yes, that is what is meant. I'll reword that, also to avoid the word cloud again, I missed that.

File: tobac/plotting.py
Comment (snilsn) [reply tobac/plotting.py:189]: Yes, I rephrased that.

File: tobac/plotting.py
Comment (snilsn) [reply tobac/plotting.py:574]: Probably, but not sure about that. It doesn't work out of the box with the tracks I created from the testing-datasets in my notebook.

File: tobac/plotting.py
Comment (snilsn) [reply tobac/plotting.py:576]: This was the first function I encountered that didn't work, but I already started writing the docstring. Probably shouldn't have included that in the PR :) 
Will remove.

File: tobac/plotting.py
Comment (snilsn) [reply tobac/plotting.py:2275]: Oh not in that case apparently. Sometimes it has a default value and sometimes it doesn't, misssed that.

**Comment by snilsn** [comment]
Thanks for your feedback, @JuliaKukulies, most of your suggestions should be covered by the last commit.

File: tobac/plotting.py
Comment (JuliaKukulies) [inline tobac/plotting.py:567]: For this function, you reversed the docstring changes. Just checking if this was intentionally?

**Review by JuliaKukulies — APPROVED** [review]
All right, approved from my side! Thanks for the changes @snilsn 

I guess we can see over that codecov fails here - maybe discuss later how to set the threshold for accepted decrease in coverage.

File: tobac/plotting.py
Comment (JuliaKukulies) [reply tobac/plotting.py:43]: OK, you are right. So maybe add it there? I think it is useful to know from the description, even though you can see that if or not a filename is one of the input parameters.

**Review by freemansw1 — APPROVED** [review]
Just a minor comment. Great work again.

**Comment by freemansw1** [comment]
OK- I've opened a new issue for the broken functions (#158).

File: tobac/plotting.py
Comment (snilsn) [reply tobac/plotting.py:204]: Yes, true, going to add that.

Comment (snilsn) [reply tobac/plotting.py:204]: Added it to the other keyword arguments that aren't actually optional in the function description and removed the 'optional'. But this feels like bad style, maybe we should delete the default values in the code for these kwargs? There should be no working code including this function without passing them anyway.

Comment (freemansw1) [reply tobac/plotting.py:203]: Yes, generally I agree with you that this is bad style. However, given our prior experience with needing to depreciate a positional argument (#125), I'm disinclined from moving a kwarg to a positional arg without a lot of discussion. I think that given the discussion in #158, we will put this on hold for now until we ultimately decide what to do with the functions.

**Comment by freemansw1** [comment]
Now that we have two approvals and this is passing non-coverage tests (see #159), I'm happy to merge this if you are happy for it to be merged @snilsn .

File: tobac/plotting.py
Comment (snilsn) [reply tobac/plotting.py:203]: Oh yes, forgot about that. I agree to put this on hold for now.

**Comment by snilsn** [comment]
Yes all good, please go ahead @freemansw1

**Pull request merged**
